functionaltest/src/test/java/TestRunner.java

```diff
@@ -0,0 +1,41 @@
+import cucumber.api.CucumberOptions;
+import cucumber.api.testng.TestNGCucumberRunner;
+import cucumber.api.testng.CucumberFeatureWrapper;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+@CucumberOptions(
+    features = "src/test/resources/features",
+    glue = {"stepdefs"},
+    tags = {"@registerTest, @loginTest, @updateTest"},
```

Comment:
Do we need to specify the tags here?

Reply:
I put them there when I was messing around with the tests. In this case it's the same to have `tags = {~@ignore}`